include/seahorn/OperationalSemantics.hh

```diff
@@ -5,6 +5,12 @@
 #include "llvm/IR/InstVisitor.h"
 #include <memory>
 
+#include "seadsa/Global.hh"
+
+#include "clam/CfgBuilder.hh"
```

Comment:
no clam include headers should be in `OperationalSemanitcs`. It is too generic and does not depend on clam